File: Go/mac_address.go
/*
    MAC address checker, Go
    Module written by Brandon Arvanaghi 
    Website: arvanaghi.com 
    Twitter: @arvanaghi
*/

package main

import (
  "fmt"
  "net"
  "strings"
)

func main() {

  EvidenceOfSandbox := make([]net.HardwareAddr, 0)

  badMacAddresses := [...]string{`00:0C:29`, `00:1C:14`, `00:50:56`, `00:05:69`, `08:00:27`}

  NICs, _ := net.Interfaces()
  for _, NIC := range NICs {
    for _, badMacAddress := range badMacAddresses {
      if strings.Contains(strings.ToLower(NIC.HardwareAddr.String()), strings.ToLower(badMacAddress)) {
        EvidenceOfSandbox = append(EvidenceOfSandbox, NIC.HardwareAddr)
      }
    }   
  }

  if len(EvidenceOfSandbox) == 0 {
    fmt.Println("Proceed!")
  } else {
    fmt.Println(EvidenceOfSandbox)
  }

}

File: C#/mac_address.cs
/*
    MAC address checker, C#
    Module written by Brandon Arvanaghi 
    Website: arvanaghi.com 
    Twitter: @arvanaghi
*/

using System;
using System.Collections.Generic;
using System.Text.RegularExpressions;
using System.Net.NetworkInformation;

namespace macAddressChecker {
    class Program
    {
        static void Main(string[] args)
        {
            List<string> EvidenceOfSandbox = new List<string>();

            string[] badMacAddresses = {@"000C29", @"001C14", @"005056", @"000569", @"080027"};

            NetworkInterface[] NICs = NetworkInterface.GetAllNetworkInterfaces();
            foreach (NetworkInterface NIC in NICs) {
                foreach (string badMacAddress in badMacAddresses) {
                    if (NIC.GetPhysicalAddress().ToString().ToLower().Contains(badMacAddress.ToLower())) {
                        EvidenceOfSandbox.Add(Regex.Replace(NIC.GetPhysicalAddress().ToString(), ".{2}", "$0:").TrimEnd(':'));
                    }
                }
            }

            if (EvidenceOfSandbox.Count == 0) {
                Console.WriteLine("Proceed!");
                Console.ReadKey();
            }
            else {
                Console.Write(string.Join("\n", EvidenceOfSandbox));
                Console.ReadKey();
            }

        }

    }
}